lib/experimental/class-wp-webfonts.php

```diff
@@ -145,28 +145,80 @@ public function register_webfont( array $webfont ) {
 	/**
 	 * Enqueue a font-family that has been already registered.
 	 *
-	 * @param string $font_family_name The font family name to be enqueued.
+	 * @param string     $font_family_name The font family name to be enqueued.
+	 * @param array|null $font_face The font face to selectively enqueue.
 	 * @return bool True if successfully enqueued, else false.
 	 */
-	public function enqueue_webfont( $font_family_name ) {
+	public function enqueue_webfont( $font_family_name, $font_face = null ) {
 		$slug = $this->get_font_slug( $font_family_name );
 
-		if ( isset( $this->enqueued_webfonts[ $slug ] ) ) {
+		// The font family is already enqueued and there are none left to enqueue.
+		if ( isset( $this->enqueued_webfonts[ $slug ] ) && ! isset( $this->registered_webfonts[ $slug ] ) ) {
 			return true;
 		}
 
 		if ( ! isset( $this->registered_webfonts[ $slug ] ) ) {
-			/* translators: %s unique slug to identify the font family of the webfont */
-			_doing_it_wrong( __METHOD__, sprintf( __( 'The "%s" font family is not registered.', 'gutenberg' ), $slug ), '6.0.0' );
+			_doing_it_wrong(
+				__METHOD__,
+				sprintf(
+					/* translators: %s unique slug to identify the font family of the webfont */
+					esc_html__( 'The "%s" font family is not registered.', 'gutenberg' ),
+					wp_kses_data( $slug )
+				),
+				'6.0.0'
+			);
 
 			return false;
 		}
 
-		$this->enqueued_webfonts[ $slug ] = $this->registered_webfonts[ $slug ];
-		unset( $this->registered_webfonts[ $slug ] );
+		// Enqueueing the font family completely.
+		if ( ! $font_face ) {
+			$font_family_to_enqueue           = $this->unregister_font_family( $font_family_name );
+			$this->enqueued_webfonts[ $slug ] = $font_family_to_enqueue;
+
+			return true;
+		}
+
+		// Enqueueing a single font face.
+
+		$font_face            = gutenberg_webfont_to_kebab_case( $font_face );
+		$font_face_to_enqueue = $this->unregister_font_face( $font_face );
+
+		if ( ! $font_face_to_enqueue ) {
+			trigger_error(
+				sprintf(
+					/* translators: %1$s: font family, %2$s: font weight, %3$s: font style */
+					esc_html__( 'The "%1$s:%2$s:%3$s" font face is not registered.', 'gutenberg' ),
+					wp_kses_data( $font_face['font-family'] ),
+					wp_kses_data( $font_face['font-weight'] ),
+					wp_kses_data( $font_face['font-style'] )
```

Comment:
Same as the other comments for `wp_kses_*` vs `esc_*`. If there are many calls like these, the simpler esc_* functions will be more performant 👍